### PR TITLE
Fix protocol-relative resolution

### PR DESCRIPTION
**Pull request by wycats**
This PR fixes protocol-relative resolution. I couldn't get the browser tests to run locally, so I'll have to let CI catch it.

I'm also not sure if I matched the expected repo's code style, so please let me know if there's anything I got wrong.

### CODE REVIEW

File: test/test.browser.js [hunk 1]
```diff
@@ -73,6 +73,9 @@ selenium.describe ('Chrome test', (done) => {
                 path.relativeToFile ('http://foo.com/scripts/bar.js', '/bar.js.map')
                     .should.equal ('file:///bar.js.map')
 
+                path.relativeToFile ('http://foo.com/scripts/bar.js', '//bar.com/bar.js.map')
+                    .should.equal ('http:///bar.com/bar.js.map')
```
Comment (xpl) [inline test/test.browser.js]: `http:///bar.com` → `http://bar.com`?

**Comment by xpl** [comment]
> I couldn't get the browser tests to run locally, so I'll have to let CI catch it.

Yeah, I couldn't run them too :) It got broken some time ago and I don't quite have the time to figure why :(

**Comment by xpl** [comment]
Ahh, there is also a flaw in the CI script, it tries to push changes from built PR (resulting in auth error), but it needs only do that when building master.

No worry about that.

**Comment by wycats** [comment]
@xpl I'd be willing to update the browser CI to either use headless chrome, jsdom or something else. Are you interested?

**Comment by xpl** [comment]
@wycats That would be super awesome!

File: test/test.browser.js (hunk 1 above)
Comment (xpl) [reply test/test.browser.js]: I'll make the fix upon merging, no worries.

**Comment by wycats** [comment]
@xpl Is there anything else I need to do to move this PR along?

**Comment by xpl** [comment]
> Is there anything else I need to do to move this PR along?

@wycats Nope, I'm going to merge it now :)

**Pull request merged**